shell/platform/android/test/io/flutter/embedding/android/FlutterActivityAndFragmentDelegateTest.java

```diff
@@ -97,18 +97,21 @@ public void itSendsLifecycleEventsToFlutter() {
     verify(mockFlutterEngine.getLifecycleChannel(), never()).appIsResumed();
     verify(mockFlutterEngine.getLifecycleChannel(), never()).appIsPaused();
     verify(mockFlutterEngine.getLifecycleChannel(), never()).appIsInactive();
+    verify(mockFlutterEngine.getLifecycleChannel(), never()).appIsDetached();
```

Comment:
seems like it's getting a bit verbose. Can we just verifyNoMoreInteractions? (it may not work though if there's more things going on in the mock but I suspect not)

Reply:
is detached will be called at the end. I think we cannot use verifyNoMoreInteractions?